pubsub/subscriptions/create_cloud_storage_subscription.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package subscriptions
+
+// [START pubsub_cloudstorage_subscription]
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+)
+
+// createCloudStorageSubscription creates a Pub/Sub subscription that exports messages to Cloud Storage.
+func createCloudStorageSubscription(w io.Writer, projectID, subID string, topic *pubsub.Topic, bucket string) error {
+	// projectID := "my-project-id"
+	// subID := "my-sub"
+	// topic of type https://godoc.org/cloud.google.com/go/pubsub#Topic
+	// note bucket should not have the gs:// prefix
```

Comment:
Don't think it's necessary to state this (or it can be done in the docs surrounding the sample block).

Reply:
I was trying to capture the docs from [the proto](https://github.com/googleapis/googleapis/blob/64d54ff78cfe7cdc2f4b95717dc9afa3ef32a29a/google/pubsub/v1/pubsub.proto#L1057), but I think you're right that it's a bit redundant since we already give the example.

Reply:
Yeah I just mention it because it's not something we generally note in the GCS docs; `gs://` is more of a gsutil construct and not used in clients or API calls elsewhere at all.